stream_alert/shared/alert.py

```diff
@@ -215,29 +228,15 @@ def output_dict(self):
             'log_source': self.log_source or '',
             'log_type': self.log_type or '',
             'outputs': list(sorted(self.outputs)),  # List instead of set for JSON-compatibility
```

Comment:
This is your code, but the `sorted` function already returns a list.. casting to list again just copies the list again..